counting_sort/counting_sort.py

```diff
@@ -0,0 +1,40 @@
+#Running time Complexity: O(n)
+#Auxillary Space: O(n)
+def count_occurrences(a, c):
+    """
+    :param a: Iterable of elements
+    :param c: Intermediate variable
+    :return: No returns, counts occurrences
+    Time Complexity: O(n)
+    Used loops: one for bins other for right indexes
+    """
+    for i in a:
+        c[a[i]] += 1
+    for i in range(1, len(c)):
+        c[i] = c[i - 1] + c[i]
+
+
+def counting_sort(a, k):
+    """
+    Input params: Array, Max value
+    Return: Output array
+    Variables : b:initialising all to zero
```

Comment:
Use same pattern for docstrings as used in `count_occurrences` method. Also, try to give meaningful name to variables.